ResearchUI/ResearchUI/iOS/RSDStepViewController.swift

```diff
@@ -71,31 +71,41 @@ public protocol RSDStepViewControllerProtocol : RSDStepController {
 /// UI architecture.
 open class RSDStepViewController : UIViewController, RSDStepViewControllerProtocol, RSDCancelActionController {
 
-    /// Pointer back to the task controller that is displaying the step controller. The implementation
-    /// of the task controller should set this pointer before displaying the step controller.
-    open weak var taskController: RSDTaskController!
+//    /// Pointer back to the task controller that is displaying the step controller. The implementation
+//    /// of the task controller should set this pointer before displaying the step controller.
+//    public var taskController: RSDTaskController! {
+//        return self.stepViewModel?.rootPathComponent?.taskController
+//    }
     
     /// The step view controller delegate.
     open weak var delegate: RSDStepViewControllerDelegate?
     
-    /// The step presented by the step view controller.
+    /// The stepViewModel presented by the step view controller.
     ///
-    /// If you use a storyboard to initialize the step view controller, `init(step:)` isn't called,
-    /// so you need to set the `step` property directly before the step view controller is presented.
+    /// If you use a storyboard to initialize the step view controller, `init(step:parent:)` isn't called,
+    /// so you need to set the `stepViewModel` property directly before the step view controller is presented.
```

Comment:
do you need to set the parent property as well?

Reply:
No, the stepViewModel carries two properties. It's parent path component **and** the step.